src/screens/Assets/BTCView.js

```diff
@@ -108,10 +110,28 @@ class BTCView extends React.Component<Props, State> {
     this.refreshBalance();
   }
 
+  onPressSend = () => {
+    const { supportedAssets } = this.props;
+    const btcToken = supportedAssets.find(e => e.symbol === BTC);
 
-  onPressSend = (assetData) => {
-    // TODO: Start send flow
-    this.props.navigation.navigate(SEND_BITCOIN_FLOW, { assetData });
+    if (!btcToken) {
+      console.error('BTC token not found');
+      return;
+    }
+
+    const {
+      symbol: token,
+      iconUrl: icon,
+      decimals,
+    } = btcToken;
+
+    const assetData: AssetData = {
+      token,
+      icon,
+      decimals,
+    };
+
+    this.props.navigation.navigate(SEND_TOKEN_FROM_ASSET_FLOW, { assetData });
```

Comment:
[SEND_BITCOIN_CONTACTS]: SendBitcoinContactsScreen,	because the one specified has ETH validation